src/NBitcoin/BlockHeader.cs

```diff
@@ -56,7 +56,6 @@ public DateTimeOffset BlockTime
             }
         }
 
-        [Obsolete("Please use the Load method outside of consensus.")]
```

Comment:
Removed on purpose because it's not obsolete if we need to use it.

Reply:
hmmm.... it is, we should only use it from the ConsensusFactory, whis should actually be internal (but we need NBitcoin to be in Stratis.Bitcoin for that)

Reply:
Yes. Instead of being obsolete it should have a remark that it should be only used from consensus factory

Reply:
I prefer you put it back so to avoid mistakes by other devs

Reply:
Makes sense. Resoted